contributing.md

```diff
@@ -0,0 +1,61 @@
+
+## Versioning
+
+We closely follow the [Semantic Versioning System](https://semver.org/)
+
+Since this is a relatively small project, we combine the idea of minor & patch versioning into simple MINOR versioning, so we only do `x.y` vs `x.y.z`
+
+So what constitutes a major version change -
+
+> MAJOR version when you make incompatible API changes
+
+For this design project, that may include:
+
+- Removing previously defined external accessible css rules
+- A large design change that would affect the entire system
+- Changing token definitions (i.e. spacing) causing many elements, components to be affected
+
+What constitutes a MINOR version change -
+
+> MINOR version when you add functionality in a backwards compatible manner, and
+> PATCH version when you make backwards compatible bug fixes.
+
+For this design project, this may include:
+
+New components, elements, css rules
+
+- We don't need to create a MINOR version for every change unless we want it to be publicly available to other applications. We can have multiple PRs included in 1 MINOR version update.
+
+### Finding versions
+
+All cut releases are listed on the [release](https://github.com/turingschool/savile/releases) page within Github. As well as live under the `css/versions` directory.
+
+### Cutting a New Version
+
+#### Cutting a new MINOR version on the current MAJOR Version (Most Common)
+1. Checkout a new branch following the name conventions `v[CURRENT_MAJOR_VERSION].[UPDATED_MINOR_VERSION]` (e.q. `v1.3`)
+2. Build the site. This can be done with `bundle exec jekyll build`
+3. Locate the `_site/css/main.css` file and copy the contents to clipboard
+4. Create a new file under `css/versions/[CURRENT_MAJOR_VERSION]/`; name the file following the naming convention `[version-number].css` (e.g `1.3.css`). Paste what is on your clipboard into this file
+5. Override the current `v[CURRENT_MAJOR_VERSION]/[CURRENT_MAJOR_VERSION]-latest.css` (e.g. `v1/1-latest.css`) with the contents of `_site/css/main.css` as well.
+6. Commit this new and updated file with `git commit -m "Cut v[version-number] release"`
+7. Push these changes up and make a PR
+8. Once merged make a new release in Github by going to https://github.com/turingschool/savile/releases and click 'Draft a new release'. Tag version (e.g v1.3),target (master) and release title (e.g 1.3), and what new features are included
+
+Once this PR is merged, two things will happen:
+- The `main` branch will be updated with these changes (as you'd expect)
+- Savile users can now choose to access this new version at `https://savile.turing.edu/css/versions/[version-file-name].css`.
+
+#### Cutting a new MAJOR version
+Similar to the instuctions above with the extra step of creating a new folder (before step 4) with the major version number under `css/versions` and rather than updating `[MAJOR_VERSION]-latest.css` you will need to create that file.
```

Comment:
```suggestion
Similar to the instructions above with the extra step of creating a new folder (before step 4) with the major version number under `css/versions` and rather than updating `[MAJOR_VERSION]-latest.css` you will need to create that file.
```